Broken, SIGSEGV

The current repo is broken...

after "go build"

```
wmortl@Discovery:/mnt/e/Code/gpython$ ./gpython
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x5986b1]

goroutine 1 [running]:
main.main()
        /mnt/e/Code/gpython/main.go:49 +0x171
```

Seems to be context related from line 49

--- a/main.go
+++ b/main.go
@@ -9,16 +9,15 @@ package main
 import (
 	"flag"
 	"fmt"
+	"log"
+	"os"
 	"runtime"
 	"runtime/pprof"
 
+	_ "github.com/go-python/gpython/modules"
+	"github.com/go-python/gpython/py"
 	"github.com/go-python/gpython/repl"
 	"github.com/go-python/gpython/repl/cli"
-
-	"log"
-	"os"
-
-	"github.com/go-python/gpython/py"
 )
 
 // Globals

